### ISSUE TITLE
VEC02 Vector Surveillance Reporting (Report detail collection in a Vector Surveillance Session

### ISSUE THREAD

**Issue by karlasomera** [EIDSS/EIDSS7#416]
**Summary**
Unable to select the add button on the Field test

**Details**
User name - AdjararegionalVetScript

**To Reproduce**
Steps to reproduce the behavior (if in scripted testing, can refer to script step):
1. Go to… Session#vsstbtbz220093
2. Click on… detail collections data
3. Select pool vector ID VSRtbtbZ220016 and select edit
4. Under the field tests unable to add (button is grayed out)

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by aatmns** [comment]
8/2/22
Unable to reproduce on the Tester's machine, with the same credentials, and the same record IDs.

Logged in as adjararegionalvet.

1 Opened the record VSSTBTBZ220093 used in the scenario.
![vector1](https://user-images.githubusercontent.com/32458032/182370644-53e12819-e1fb-4512-9fd4-b427db864369.png)

2 Opened the Detailed Collection page:
![vector2](https://user-images.githubusercontent.com/32458032/182370753-88746695-5f96-4946-bd3a-cadc7f39b053.png)

3 Navigated to Field Tests page.
The "Add" control is enabled.
![vector5](https://user-images.githubusercontent.com/32458032/182371043-7d4f577c-e81e-4160-ba0c-7961fb401483.png)

4 Added a field test record successfully:
![vector3](https://user-images.githubusercontent.com/32458032/182371133-a1f7fc20-4220-4e59-90ad-9453fbd09059.png)

5. Successfully saved the field test record:
![vector4](https://user-images.githubusercontent.com/32458032/182371202-1980b6fd-145c-4e28-a169-16d31c3233ca.png)

The system operated as designed.

**Comment by karlasomera** [comment]
After retesting this issue, the add field test button is available on the external environment.  When working in the GAT environment I am able to proceed to section 8 when I am unable to add a sample type  in the sample section.  Unable to move forward.   in addition, If I select the review section to save it does not allow me to save.  (This is a known issue in GAT environment).  Unable to close due to waiting to retest on the GAT environment. 
I used the following User:  Adjararegionalvet.  

I have attached a screen shot for your reference.  Unable to save to move forward with this specific script.

![Capture 3_VEC02_adding sample_KPS](https://user-images.githubusercontent.com/104407068/182442285-88b025bd-35ce-45c3-aab8-a0264f98ecdd.PNG)
